pyHS100/pyHS100.py

```diff
@@ -305,6 +344,39 @@ def location(self):
 
         return loc
 
+    @location.setter
+    def location(self, location=(0.0, 0.0)):
+        """
+        Sets new location
+
+        :param tuple location: (float(latitude), float(latitude))
+        :raises SmartPlugException: on error
+        """
+
+        latitude, longitude = location
+
+        if latitude < 0:
+            latitude = -latitude
+        if longitude < 0:
+            longitude = -longitude
+
```

Comment:
my goof on that one. my intention was to make the positive ones negative. tho have not done to much research into lat and lon. but i do know that some times my lat and lon have come up either positive or negative and only the negative ones are correct. I went based off of that. is there even such a thing as a positive lat and lon? I do not know